geode-core/src/distributedTest/java/org/apache/geode/cache30/GlobalRegionDUnitTest.java

```diff
@@ -240,80 +206,66 @@ public void uncaughtException(Thread t, Throwable e) {
         };
 
         Thread[] threads = new Thread[threadsPerVM];
-        for (int i = 0; i < threadsPerVM; i++) {
-          Thread thread = new Thread(group, new Runnable() {
-            @Override
-            public void run() {
+        for (int i1 = 0; i1 < threadsPerVM; i1++) {
+          Thread thread = new Thread(group, () -> {
+            try {
+              final Random rand = new Random(System.identityHashCode(this));
               try {
-                final Random rand = new Random(System.identityHashCode(this));
-                try {
-                  Region<Object, Integer> region = getRootRegion().getSubregion(name);
-                  for (int j = 0; j < incrementsPerThread; j++) {
-                    Thread.sleep(rand.nextInt(30) + 30);
-
-                    Lock lock = region.getDistributedLock(key);
-                    assertThat(lock.tryLock(-1, TimeUnit.MILLISECONDS)).isTrue();
-
-                    Integer value = region.get(key);
-                    Integer oldValue = value;
-                    if (value == null) {
-                      value = 1;
-
-                    } else {
-                      Integer v = value;
-                      value = v + 1;
-                    }
-
-                    assertThat(oldValue).isEqualTo(region.get(key));
-                    region.put(key, value);
-                    assertThat(value).isEqualTo(region.get(key));
-
-                    lock.unlock();
+                Region<Object, Integer> region = getRootRegion().getSubregion(name);
+                for (int j = 0; j < incrementsPerThread; j++) {
+                  Thread.sleep(rand.nextInt(30) + 30);
+
+                  Lock lock = region.getDistributedLock(key);
+                  assertThat(lock.tryLock(-1, TimeUnit.MILLISECONDS)).isTrue();
+
+                  Integer value = region.get(key);
+                  Integer oldValue = value;
+                  if (value == null) {
+                    value = 1;
+
+                  } else {
+                    Integer v = value;
+                    value = v + 1;
                   }
 
-                } catch (IllegalStateException | InterruptedException | CacheLoaderException
-                    | CacheWriterException | TimeoutException ex) {
-                  fail("While incrementing", ex);
+                  assertThat(oldValue).isEqualTo(region.get(key));
+                  region.put(key, value);
+                  assertThat(value).isEqualTo(region.get(key));
+
+                  lock.unlock();
                 }
-              } catch (VirtualMachineError e) {
-                throw e;
-              } catch (Throwable t) {
-                logger
-                    .info("testSynchronousIncrements." + this + " caught Throwable", t);
+
+              } catch (IllegalStateException | InterruptedException | CacheLoaderException
```

Comment:
Optional: Remove catch block containing fail and add throws clause to method(s).

Reply:
done